packages/documentation-site/patternfly-docs/content/accessibility/page/page.md

```diff
@@ -3,30 +3,81 @@ id: Page
 section: components
 ---
 
-The **page** component is used to define the basic layout of a page with either vertical or horizontal navigation. Page layouts are defined using page sections, such as the header, body, and footer of a page. The basic layout of a page depends on whether your page uses vertical or horizontal navigation.
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
 
-A page can be combined with many other components, so to create an accessible page, refer to the accessibility recommendations for each component.
+## Accessibility
 
-**Keyboard users** should be able to navigate through the masthead of the page and into other interactive elements of the page using **Tab** and **Shift + Tab** to move forward and backward through interactive elements. 
+To implement an accessible PatternFly **page**:
 
-**Screen reader users** should be able to navigate through and interact with the masthead of the page using the same interactions as keyboard users. Elements of a page that do not have visible descriptive text, such as toggle icons, should have alternative text for screen readers.
+- Provide a [skip to content](/components/skip-to-content) if more than one page shares the same heading or sidebar content.
+- Give each `nav` element on the page a unique `aria-label`.
```

Comment:
Ah good catch, agreed we should be consistent in the verbiage.

Though now I'm slightly conflicted, in that I'm wondering if we should say that a nav should have a unique label regardless of amount, i.e. whether there's 7 or only 1. I guess I'm thinking in terms of rotor menus/wherever a list of X element types are listed out (lists, headings, etc). Even with only 1 nav (or list, or whatever) on a page, I feel like providing an aria-label can only help, whereas omitting one could possibly hurt?

For example, if the nav is announced as just "nav' to a user, it could be site navigation, page navigation, breadcrumb navigation, etc. So what the user would be jumping to could be a bit of an unknown.

What do you think, though?

Reply:
I think it definitely could be helpful, but I'm wondering if it's fully required. I wonder if we could explain how it can be helpful but say that it's only required when there's multiple? (And maybe we could even give the examples you just stated.)

Reply:
Maybe, "If there are multiple `nav` elements on the page, give each one a unique `aria-label`." as the checkbox label, then a description of, "You should also consider providing an `aria-label` when there's only a single `nav` on the page, if the purpose of that nav is not the primary navigation."

Or something along those lines? Not sure how I feel about "primary navigation"; "primary site navigation", really just wanting to convey "something that isn't the typical navigation for a page"

Reply:
Yeah, I think that makes sense! And I think calling it a primary navigation isn't inherently bad. You could even work with those words a bit between both and say something like, "You should also consider providing an `aria-label` when there's only a single nav on the page, if the purpose of that nav isn't a typical, primary navigation for a page." (To be honest though, I think all of these options are probably fine though and hopefully get the point across.)